### DISCUSSION TITLE
graphql/scalars/docs/scalars/void

### DISCUSSION THREAD

**Discussion by giscus** [Urigo/graphql-scalars#1971] [Docs Discussions]
# graphql/scalars/docs/scalars/void

GraphQL Scalars Documentation

https://the-guild.dev/graphql/scalars/docs/scalars/void

**Comment by minaBonyadi** [comment]
Hi my my Graphql service return Void response. but I found out graphql schema cannot handle Void variable. Do you know how shouldI have this type in my response? the code is like this: 
type Query {
    updateEmailAddress(id: String, emailAddress: String): Void }

**Reply by ardatan** [reply]
Sorry I couldn't follow. Could you share a reproduction on CodeSandbox? Thanks!

**Comment by minaBonyadi** [comment]
this is my service 
@QueryMapping
    public void updateEmailAddress(@Argument String id, @Argument String emailAddress) {
        orderGateway.updateEmailAddress(id, emailAddress);
    }

**Reply by minaBonyadi** [reply]
and I want to call this rest service throw the graphql

**Reply by minaBonyadi** [reply]
my graphql schema is like this :

scalar Date
scalar DateTime
scalar Void

type Query {
    updateEmailAddress(id: String, emailAddress: String): Void
}

**Reply by ardatan** [reply]
Sorry these are not helpful. Could you create a reproduction on CodeSandbox?

**Comment by babak-behravesh-bestsecret** [comment]
I faced the same issue before. graphql-scalars package should help to fix the issue. https://stackoverflow.com/questions/44737043/is-it-possible-to-not-return-any-data-when-using-a-graphql-mutation/44773532#44773532
